### PR TITLE
Addingjapanesesupport

### PR DESCRIPTION
**Pull request by jdomaga**
japanese language support, using iso language code ja-JP as per https://docs.microsoft.com/en-us/previous-versions/commerce-server/ee796272(v=cs.20) .

### CODE REVIEW

File: source/IdentityServer3.Localization/IdentityServer3.Localization.csproj [hunk 1]
```diff
@@ -81,6 +81,10 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Resources\Events\Events.ja-JP.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Events.ja-JP.Designer.cs</LastGenOutput>
```
Comment (CrescentFresh) [inline source/IdentityServer3.Localization/IdentityServer3.Localization.csproj]: The other `<EmbeddedResource>` tags do not include xml children, wondering if you should do the same.

Comment (jdomaga) [reply source/IdentityServer3.Localization/IdentityServer3.Localization.csproj]: ooh good call, fixed!

**Comment by johnkors** [comment]
Deployed as https://www.nuget.org/packages/IdentityServer3.Localization/0.3.200

**Comment by johnkors** [comment]
Added to docs: http://johnkors.github.io/IdentityServer3.Contrib.Localization/#/ja-JP

**Pull request merged**